cfme/rest/gen_data.py

```diff
@@ -403,3 +417,28 @@ def _finished():
             logger.warning("Failed to delete arbitration settings.")
 
     return response
+
+
+def orchestration_templates(request, rest_api, num=2):
+    collection = rest_api.collections.orchestration_templates
+    body = []
+    for _ in range(num):
```

Comment:
Consider using the number here in a meaningful way by including it in the name somewhere - for example test_{num}_{unique}